### PR TITLE
Feature/47153 census ward removal

### PR DESCRIPTION
**Pull request by superfurryadamal**
Removed all references to CASWard as the field has now been archived

### CODE REVIEW

**Comment by jonbanner** [comment]
@superfurryadamal looks good - i think there is some JS for the suggestion component on this field that can be removed

Should just be removing the config in Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js

**Comment by superfurryadamal** [comment]
> @superfurryadamal looks good - i think there is some JS for the suggestion component on this field that can be removed
> 
> Should just be removing the config in Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js

thanks @jonbanner , i'll take a look

File: Web/Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js [hunk 1]
```diff
@@ -20,12 +20,6 @@ const locationAutoSuggest = (function() {
       hiddenFieldId: 'ParliamentaryConstituencyIdHidden',
       maxItems: 10
     },
-    {
-      textFieldId: 'CASWardName',
-      suggestions: window.caswards,
-      hiddenFieldId: 'CASWardIdHidden',
-      maxItems: 30
-    },
```
Comment (superfurryadamal) [inline Web/Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js:28]: @jonbanner - looks like i already removed it... was there somewhere else you mean with the JS?

Comment (jonbanner) [reply Web/Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js:28]: ah sorry missed that when looking it over.

Comment (superfurryadamal) [reply Web/Edubase.Web.UI/Assets/Scripts/GiasEdit/locationAutoSuggests.js:28]: easy done. I tried search the assets for "cas" references, so i'm hoping i caught everything

**Comment by jonbanner** [comment]
looks good to me @superfurryadamal

**Comment by superfurryadamal** [comment]
Comments have been resolved so merging into dev

**Pull request merged**